.travis.yml

```diff
@@ -57,13 +48,8 @@ install:
   - curl -O https://cdn.azul.com/zulu/bin/zulu-repo_1.0.0-3_all.deb
   - sudo apt-get -yq install ./zulu-repo_1.0.0-3_all.deb
   - sudo apt-get -q update
-  - sudo apt-get -yq install zulu17-jdk
-  - export JAVA_HOME=/usr/lib/jvm/zulu17
-  - echo "USE mysql;\nALTER USER 'root'@'localhost' IDENTIFIED BY 'mysql';\n" | mysql -u root
-  - mysql -u root -pmysql -e 'CREATE DATABASE IF NOT EXISTS `fineract_tenants`;'
-  - mysql -u root -pmysql -e 'CREATE DATABASE IF NOT EXISTS `fineract_default`;'
-# Hardcoding the time zone is a temporary fix for https://issues.apache.org/jira/browse/FINERACT-723
-  - export TZ=Asia/Kolkata
+  - sudo apt-get -yq install zulu11-jdk
+  - export JAVA_HOME=/usr/lib/jvm/zulu11
 
```

Comment:
This looks like it's still pointing to JDK 11?

Reply:
ah... didn't see that... that stuff is not necessary at all

Reply:
Fixed